src/lib/colourType.js

```diff
@@ -12,4 +13,4 @@ export const isHex = colour => {
 
 export const isRGBorHSL = colour => /^(rgb|hsl)a?\(.*?\)/.test(colour);
 
-export const isKeyword = colour => has(colourNames, colour.toLowerCase());
+export const isKeyword = colour => has.call(colourNames, colour.toLowerCase());
```

Comment:
joyent both has never had anything to do with npm and no longer controls node. You'd need to scrape npm itself for that.